app/views/groups/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Categories List</h1>
+<ul>
+  <%= render @groups %>
```

Comment:
- [ ] Please make sure to follow the [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
 - Colors
 - Typography: font face, size and weight.
 - Layout: composition and space between elements
### Your Design
![image](https://github.com/mohamedSabry0/inqezWallet/assets/45460192/a28a8ae2-6cfb-482a-9b71-e864a9803ac4)
### Required
![image](https://github.com/mohamedSabry0/inqezWallet/assets/45460192/7297eafb-c15c-4869-8c12-5c878257ea4f)
  - The "Add a new category button" should be at the bottom of the page with the background colour being `#5fb523` and a white rectangular boundary around it.
  - The listed category should have the exact design and alignment as in the original design.
  - The title "Category List" should be at the top of the page as seen in the original design.